src/console/dlt-control.c

```diff
@@ -111,6 +112,22 @@ typedef struct {
     DltFilter filter;
 } DltReceiveData;
 
+void wait_send_buffer_empty(int fd)
+{
+#ifdef __linux__
+    /* Wait until send buffer is empty */
+    while (1)
+    {
+        int unsent;
+        ioctl(fd, TIOCOUTQ, &unsent);
+        if (!unsent)
+        {
+            break;
+        }
+        usleep(100000);
```

Comment:
Timeout should be considered. There could be a situation where the user set timeout with -t option but enter this condition. It should be either put together with timeout or introduce a new option which should be isolated  (or ignore) from -t option.

Reply:
@jardous I guess you also have the another patch for adding timeout in your internal branch. It may be able to the answer to ssugiura's comment.

Reply:
@ysk-sato I could find find any PS that adds related timeout to the dlt-control. Could you, please point me, which PS did you mean?

Reply:
The title would be "dlt-control: Fix infinite loop issue at waiting send buffer empty". Could you find again?